var/spack/repos/builtin/packages/munge/package.py

```diff
@@ -10,8 +10,12 @@
 class Munge(AutotoolsPackage):
     """ MUNGE Uid 'N' Gid Emporium """
     homepage = "https://code.google.com/p/munge/"
-    url      = "https://github.com/dun/munge/releases/download/munge-0.5.11/munge-0.5.11.tar.bz2"
+    url      = "https://github.com/dun/munge/releases/download/munge-0.5.13/munge-0.5.13.tar.xz"
 
+    version('0.5.13', '9204f34aac7f0cc50880196f4a8f5f33',
+            url='https://github.com/dun/munge/releases/download/munge-0.5.13/munge-0.5.13.tar.xz')
+    version('0.5.12', '84ffef069af93caea4b4c06f183b99c0',
+            url='https://github.com/dun/munge/releases/download/munge-0.5.12/munge-0.5.12.tar.xz')
```

Comment:
These two version-specific URLs shouldn't be needed, correct?

Reply:
Right you are @adamjstewart. I'm not clear on the best way to address this so I am submitting a new pull.